### PR TITLE
Migrate schedule fix

### PR DESCRIPTION
**Pull request by AsterNighT**
### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Review by fewdan — APPROVED** [review]
LGTM

**Comment by YangKeao** [comment]
You have run this tools on examples? Cool :cool:

**Review by YangKeao — APPROVED** [review]
LGTM

**Comment by AsterNighT** [comment]
Why?

File: config/crd/bases/chaos-mesh.org_dnschaos.yaml [hunk 1]
```diff
@@ -106,15 +133,20 @@ spec:
                 nodeSelectors:
                   additionalProperties:
                     type: string
-                  description: Map of string keys and values that can be used to select nodes. Selector which must match a node's labels, and objects must belong to these selected nodes.
+                  description: Map of string keys and values that can be used to select
```
Comment (YangKeao) [inline config/crd/bases/chaos-mesh.org_dnschaos.yaml:136]: Please don't wrap the YAML file 😢

**Review by YangKeao — CHANGES_REQUESTED** [review]

**Review by YangKeao — APPROVED** [review]
LGTM

**Comment by YangKeao** [comment]
@fewdan Your LGTM has been revoked.

**Review by STRRL — APPROVED** [review]
LGTM!

**Pull request merged**